Jenkinsfile

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env groovy
+
+REPOSITORY = 'signonotron2'
+DEFAULT_SCHEMA_BRANCH = 'deployed-to-production'
+DB_ADAPTERS = ['mysql', 'postgresql']
+
+node {
+  def govuk = load '/var/lib/jenkins/groovy_scripts/govuk_jenkinslib.groovy'
+
+  properties([
+    buildDiscarder(
+      logRotator(
+        numToKeepStr: '50')
+      ),
+    [$class: 'RebuildSettings', autoRebuild: false, rebuildDisabled: false],
+    [$class: 'ThrottleJobProperty',
+      categories: [],
+      limitOneJobWithMatchingParams: true,
+      maxConcurrentPerNode: 1,
+      maxConcurrentTotal: 0,
+      paramsToUseForLimit: 'Apps using "signonotron2_test" database',
+      throttleEnabled: true,
+      throttleOption: 'category'
+    ],
+  ])
+
+  try {
+    if (!govuk.isAllowedBranchBuild(env.BRANCH_NAME)) {
```

Comment:
Thanks @suzannehamilton the `DEFAULT_SCHEMA_BRANCH` is redundant, I'm not so sure about the throttling config as old CI has configuration like this:
![screenshot from 2017-01-05 15 35 44](https://cloud.githubusercontent.com/assets/93511/21686448/61f2e908-d35d-11e6-9d14-22fd4755ef6e.png)